src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -265,6 +265,7 @@ export default class GroupComparisonPage extends React.Component<
                             <GroupComparisonPathwayMapper
                                 genomicData={
                                     this.store.alterationEnrichmentRowData
+                                        .result!
```

Comment:
probably safer to return an empty array here
```suggestion
                                        .result || []
```